pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -463,9 +463,12 @@ type SparkPodSpec struct {
 	// Tolerations specifies the tolerations listed in ".spec.tolerations" to be applied to the pod.
 	// +optional
 	Tolerations []apiv1.Toleration `json:"tolerations,omitempty"`
-	// SecurityContenxt specifies the PodSecurityContext to apply.
-	// +optional
-	SecurityContenxt *apiv1.PodSecurityContext `json:"securityContext,omitempty"`
+	// PodSecurityContenxt specifies the PodSecurityContext to apply.
+	// Optional.
```

Comment:
We should use `+optional` here and below.

Reply:
right, fixing, is this just new syntax in v1beta2 compare to Optional.?

Reply:
Some of the controller-runtime and kubebuilder tools respect this annotation.